examples/rbac/nginx/nginx-ingress-controller-rbac.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nginx-ingress
+---  
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-ingress-serviceaccount
```

Comment:
I thought about changing this, but I'm reluctant and here's why:  

Because this is an example, I think it should be clear what maps to what without having to do any mental gymnastics.  The ClusterRole, namespace, ServiceAccount, Role, RoleBinding, and ClusterRoleBinding can all drop the `-thingtype`  extension, but then we'd have 6 objects all named `nginx-ingress` and it might cause confusion to people who are making modifications to the example in their own systems.  While it's a bit of extra text and seems obvious, it's a little more clear for someone new.